### DISCUSSION TITLE
Emulating PoleMaster FOV in Stellarium

### DISCUSSION THREAD

**Discussion by jlg84** [Stellarium/stellarium#1866] [Q&A] [answered]
Is there anyone out there who can help me set up an Ocular (or whatever) to emulate the field of view when using a PoleMaster? I know it's got a 11x6-degree FOV, and I can find the chip size (it that matters) but I don't know the other specs, nor how exactly to set it up within the program. Any help would be much appreciated!

**Comment by alex-w** [comment, marked as answer]
Please check page 7 - http://www.ioptron.com/v/Manuals/3338_PoleMaster_Instruction.pdf
